.github/workflows/deliverables-push-main.yml

```diff
@@ -63,9 +63,9 @@ jobs:
     secrets: inherit
     with:
       tag: main
-      esplora_endpoint: http://35.189.57.114:3000
-      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@35.189.57.114:9045
+      esplora_endpoint: http://34.32.0.52:3000
+      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
       coordinator_port_http: 80
```

Comment:
:mega: I think this needs to change too.

Reply:
Thanks, it should be all on port `80` again.